gfortran_utils.sh

```diff
@@ -29,6 +29,32 @@ function get_distutils_platform {
     echo "macosx_${target}_${plat}"
 }
 
+function get_distutils_platform_ex {
```

Comment:
Where do you use this function?

Reply:
Can you write some tests for the function?

Reply:
used new function, added tests for old (to be deprecated) function